packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -135,9 +170,14 @@ def test_vip(dcos_api_session, container: Container, vip_net: Network, proxy_net
         raise AssertionError('\n******************************************\n'.join(failure_stack))
 
 
-def setup_vip_workload_tests(dcos_api_session, container, vip_net, proxy_net):
+def setup_vip_workload_tests(dcos_api_session, pod, container, vip_net, proxy_net):
     if len(dcos_api_session.all_slaves) < 2:
         pytest.skip('must have more than one agent for this test!')
+    if pod:
```

Comment:
Please just make a whole new method. you can copy the form and perhaps split out some shared code, but this is not a good pattern to add if statements through a previously concise function. If you are adding pod here, then this app better support all the container possibilities or at least state which ones are supported.